recipes/sup.rb

```diff
@@ -0,0 +1 @@
+hab_sup 'default'
```

Comment:
This is for convenience, so an end user can include the recipe or use the resource. Is it that bad to have it available?